server/server/data_loader/generate_sitemap.py

```diff
@@ -53,6 +53,24 @@ def create_url(text):
 
 
 def generate_sitemap(db):
+
+    bilara_texts = list(
+        db.aql.execute(
+            '''
+        FOR doc IN sc_bilara_texts
+            FILTER 'root' IN doc.muids OR 'translation' IN doc.muids
+            RETURN {
+                _key: doc._key,
+                uid: doc.uid,
+                lang: doc.lang,
+                author_uid: doc.muids[2],
+                name: null,
```

Comment:
`null` is this ok?

Reply:
I tried to read name from the names collection at the same time, but because sc_bilara_texts had more records, the query time actually exceeded expectations, and I saw that the 'name' field was not used in this method, so it was temporarily set to Null, and so on needed to be populated later.